apps/dh/api-dh/source/DataHub.WebApi/Program.cs

```diff
@@ -49,18 +51,20 @@
     options.KnownProxies.Clear();
 });
 
-services.AddHealthChecks();
-
 services.AddHttpContextAccessor();
 
 services.AddSwagger();
 
-var mitIdExternalOpenIdUrl = configuration.GetValue<string>("MITID_EXTERNAL_OPEN_ID_URL") ?? "-";
-var externalOpenIdUrl = configuration.GetValue<string>("EXTERNAL_OPEN_ID_URL") ?? "-";
-var internalOpenIdUrl = configuration.GetValue<string>("INTERNAL_OPEN_ID_URL") ?? "-";
-var backendBffAppId = configuration.GetValue<string>("BACKEND_BFF_APP_ID") ?? "-";
+// HACK: Support for 'dotnet tool swagger' execution, which requires no exceptions in Startup file.
+if (configuration.GetChildren().All(section => section.Key != UserAuthenticationOptions.SectionName))
+{
+    builder.Configuration[$"{UserAuthenticationOptions.SectionName}:{nameof(UserAuthenticationOptions.MitIdExternalMetadataAddress)}"] = "https://datahub.dk";
+    builder.Configuration[$"{UserAuthenticationOptions.SectionName}:{nameof(UserAuthenticationOptions.ExternalMetadataAddress)}"] = "https://datahub.dk";
+    builder.Configuration[$"{UserAuthenticationOptions.SectionName}:{nameof(UserAuthenticationOptions.InternalMetadataAddress)}"] = "https://datahub.dk";
+    builder.Configuration[$"{UserAuthenticationOptions.SectionName}:{nameof(UserAuthenticationOptions.BackendBffAppId)}"] = "https://datahub.dk";
+}
```

Comment:
suggestion: noticed your PR in `dh3-dev-secrets`. Should `appsettings.json.sample` (in this repo) be updated as well?